gnomad/sample_qc/pipeline.py

```diff
@@ -238,11 +239,12 @@ def annotate_sex(
         - Y_karyotype (str): Sample's chromosome Y karyotype.
         - sex_karyotype (str): Sample's sex karyotype.
 
-    :param mt: Input MatrixTable
+    :param mtds: Input MatrixTable or VariantDataset
     :param bool is_sparse: Whether input MatrixTable is in sparse data format
     :param excluded_intervals: Optional table of intervals to exclude from the computation.
     :param included_intervals: Optional table of intervals to use in the computation. REQUIRED for exomes.
     :param normalization_contig: Which chromosome to use to normalize sex chromosome coverage. Used in determining sex chromosome ploidies.
+    :param reference_genome: Reference genome used for constructing interval list. Default: 'GRCh38'
```

Comment:
I wonder if we should just infer this from the MT? example: https://github.com/broadinstitute/gnomad_methods/blob/978b57762108d6ccbe979ed7a867ffe361d146e6/gnomad/utils/filtering.py#L133